zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -283,44 +288,17 @@ void run() throws IOException, InterruptedException {
         if (cl.getCommand() == null) {
             System.out.println("Welcome to ZooKeeper!");
 
-            boolean jlinemissing = false;
-            // only use jline if it's in the classpath
-            try {
-                Class<?> consoleC = Class.forName("jline.console.ConsoleReader");
-                Class<?> completorC = Class.forName("org.apache.zookeeper.JLineZNodeCompleter");
-
-                System.out.println("JLine support is enabled");
```

Comment:
we still need to retain this line output?

Reply:
Yes. We need to retain this line output for the BSD license. Thank you for reminding me of that, in fact, I didn't realize it in the past. This line output was added now in commit https://github.com/apache/zookeeper/pull/1458/commits/16010357021235e9e32b21fe38b3ba2192490630.